src/components/custom/_dhero.tsx

```diff
@@ -267,6 +281,17 @@ const DHero = ({
                                 <Localize translate_text="Create free demo account" />
                             </DemoButton>
                         ))}
+                    {go_to_live_demo && (
```

Comment:
Can we refactor this variable, booleans should be have `is` or `has` I think the proper name for this is `is_live_demo`

Reply:
sure @prince-deriv